app/googleanalytics/pixels.py

```diff
@@ -1,35 +1,53 @@
-from urllib.parse import urlencode
-
+import os
 from flask import current_app
+from urllib.parse import quote
+
+TEST_DOMAIN = 'https://test-api.va.gov/vanotify/'
+DEV_DOMAIN = 'https://dev-api.va.gov/vanotify/'
+PERF_DOMAIN = 'https://sandbox-api.va.gov/vanotify/'
+STAGING_DOMAIN = 'https://staging-api.va.gov/vanotify/'
+PROD_DOMAIN = 'https://api.va.gov/vanotify/'
+
+NOTIFICATION_API_GA4_GET_ENDPOINT = 'ga4/open-email-tracking'
+
+GA4_PIXEL_TRACKING_NAME = 'email_open'
+GA4_PIXEL_TRACKING_SOURCE = 'vanotify'
+GA4_PIXEL_TRACKING_MEDIUM = 'email'
+
 
+def get_domain_for_environment():
```

Comment:
I debated moving this into a helper file outside of the ga4 folder. But we are not using this functionality elsewhere so I'm hesitant to move this.

Reply:
Please add the return type annotation here.

Reply:
Updated.